content/guides/tasks/create.md

```diff
@@ -0,0 +1,25 @@
+---
+rank: 2
+related_endpoints:
+  - post_tasks
+related_guides:
```

Comment:
Please fix all frontmatter where these are not an actual list or an explicit empty list `[]`

Reply:
fix't